qt_driver.py

```diff
@@ -1224,14 +1228,15 @@ def _on_save(self, do_screenshot=False):
         else:
             # This is the simple approach to set the filename, but doesn't allow
             # us to update the working_dir, if the user changes it.
-            #filename = QtGui.QFileDialog.getSaveFileName(self, 'Save file', \
+            #filename = QtWidgets.QFileDialog.getSaveFileName(self, 'Save file', \
```

Comment:
Leave this change...this commented-out code is part of a comment.

Reply:
Right. the point is that QFileDialog is not a part of QtGui in QT5
I changed the comment to get the code works in case someone uncoment it.